### PR TITLE
Change lexical scope receiver and failsafe

### PR DESCRIPTION
**Pull request by haslinghuis**

### CODE REVIEW

File: src/js/tabs/failsafe.js [hunk 1]
```diff
@@ -254,9 +254,7 @@ TABS.failsafe.initialize = function (callback, scrollPosition) {
 
         // set stage 2 failsafe procedure
         $('input[type="radio"].procedure').change(function () {
-            var element = $(this),
-                checked = element.is(':checked'),
-                id = element.attr('id');
+            element = $(this);
```
Comment (mikeller) [inline src/js/tabs/failsafe.js:257]: This really should not set the closure's `element` variable.

File: src/js/tabs/receiver.js [hunk 2]
```diff
@@ -309,7 +308,7 @@ TABS.receiver.initialize = function (callback) {
         });
 
         $("a.sticks").click(function() {
-            var
+            const
```
Comment (mikeller) [inline src/js/tabs/receiver.js:311]: Weird line break.

File: src/js/tabs/receiver.js [hunk 3]
```diff
@@ -475,25 +474,25 @@ TABS.receiver.initialize = function (callback) {
...
             }
 
-            var samples = 0,
-                svg = d3.select("svg"),
-                RX_plot_e = $('#RX_plot'),
-                margin = {top: 20, right: 0, bottom: 10, left: 40},
-                width, height, widthScale, heightScale;
+            let samples = 0;
+            const svg = d3.select("svg");
+            const rxPlotE = $('#RX_plot');
```
Comment (mikeller) [inline src/js/tabs/receiver.js:489]: A lot of code in configurator uses the `_e` suffix to denote DOM elements - not sure we want to change this.

**Review by mikeller — CHANGES_REQUESTED** [review]

**Comment by haslinghuis** [comment]
Thanks. Fixed as requested per review.

**Review by mikeller — APPROVED** [review]

**Pull request merged**